Server/src/quadradiusr_server/game.py

```diff
@@ -1,28 +1,87 @@
+import base64
+import json
 from dataclasses import dataclass
 from typing import Dict, Tuple
 
+import jsondiff as jsondiff
+
 from quadradiusr_server.constants import QrwsCloseCode
 from quadradiusr_server.db.base import Game, clone_db_object
+from quadradiusr_server.db.base import User
+from quadradiusr_server.db.database_engine import DatabaseEngine
 from quadradiusr_server.db.repository import Repository
-from quadradiusr_server.qrws_connection import BasicConnection
-from quadradiusr_server.qrws_messages import Message, KickMessage
+from quadradiusr_server.notification import NotificationService
+from quadradiusr_server.qrws_connection import BasicConnection, QrwsConnection
+from quadradiusr_server.qrws_messages import Message, KickMessage, GameStateMessage
 
 
 @dataclass
 class Square:
```

Comment:
Don't we want to keep the same naming as in client and original? If so rename to Tile pls